### ISSUE TITLE
linux3.16: WiFi broken

### ISSUE THREAD

**Issue by cr6git** [void-linux/void-packages#1262]
### System

* xuname:  
````
Void 3.16.57_1 x86_64 GenuineIntel uptodate rrrmFFFFFFFFFFFFFFFFF
````

* package(s):  
````
[*] linux3.16-3.16.57_1
[*] NetworkManager-1.10.6_2
[*] network-manager-applet-1.8.10_2
````

### Expected behavior
I should be able to connect to my WiFi network.
### Actual behavior
Doesn't work at all. `network-manager-applet` cannot even detect any wireless network...

````
lspci |grep Network
02:00.0 Network controller: Intel Corporation Centrino Wireless-N 1000 [Condor Peak]

lsmod |grep wifi
iwlwifi                95957  1 iwldvm
cfg80211              442249  3 iwlwifi,mac80211,iwldvm

dmesg |grep iwlwifi
[    9.057562] iwlwifi 0000:02:00.0: can't disable ASPM; OS doesn't have ASPM control
[    9.057765] iwlwifi 0000:02:00.0: irq 43 for MSI/MSI-X
[    9.118878] iwlwifi 0000:02:00.0: loaded firmware version 39.31.5.1 build 35138 op_mode iwldvm
[    9.193704] iwlwifi 0000:02:00.0: CONFIG_IWLWIFI_DEBUG disabled
[    9.193710] iwlwifi 0000:02:00.0: CONFIG_IWLWIFI_DEBUGFS disabled
[    9.193713] iwlwifi 0000:02:00.0: CONFIG_IWLWIFI_DEVICE_TRACING disabled
[    9.193717] iwlwifi 0000:02:00.0: Detected Intel(R) Centrino(R) Wireless-N 1000 BGN, REV=0x6C
[    9.193817] iwlwifi 0000:02:00.0: L1 Enabled - LTR Disabled
[   19.193803] iwlwifi 0000:02:00.0: L1 Enabled - LTR Disabled
[   19.201073] iwlwifi 0000:02:00.0: Radio type=0x0-0x0-0x3
[   19.232817] iwlwifi 0000:02:00.0: L1 Enabled - LTR Disabled
[   19.240097] iwlwifi 0000:02:00.0: Radio type=0x0-0x0-0x3
[   20.902393] iwlwifi 0000:02:00.0: L1 Enabled - LTR Disabled
[   20.909650] iwlwifi 0000:02:00.0: Radio type=0x0-0x0-0x3
[   20.951438] iwlwifi 0000:02:00.0: L1 Enabled - LTR Disabled
[   20.958722] iwlwifi 0000:02:00.0: Radio type=0x0-0x0-0x3
````

**Comment by cr6git** [comment]
@pullmoll does it work for you?

**Comment by pullmoll** [comment]
I did not test the kernel. I can try on my laptop which has a WIFI adapter later.
You know you have to disable `/etc/sv/dhcpcd` to get WIFI working with NetworkManager?

**Comment by cr6git** [comment]
I know how to use them, no problem ;)
Same problem with `dhcpcd`

**Comment by pullmoll** [comment]
Works for me on my Lenove GS505 laptop with Qualcomm Atheros AR9485 WIFI adapter.
Looks like it's iwlwifi specific then...

**Comment by pullmoll** [comment]
Does not work on a Toshiba Satellite laptop with `broadcom-wl-dkms`. There I see a missing file `brcm/BCM43142A0-0930-0225.hcd` listed in dmesg. I'll try to get this one and see if it works afterwards.
Note: The laptop works with the dkms module and kernel 4.17

**Comment by pullmoll** [comment]
@cr6git Since you have a machine to test this on, by all means: give it a try.
There will be very few people using 3.6.x anyway.

**Comment by cr6git** [comment]
fixed in 3.16.59 :sweat_smile:
